package.json

```diff
@@ -23,7 +23,7 @@
     "check:change": "beachball check --scope \"!packages/fluentui/*\"",
     "check:modified-files": "yarn workspace @fluentui/scripts just check-for-modified-files",
     "check:affected-package": "node ./scripts/monorepo/checkIfPackagesAffected.js",
-    "check:installed-dependencies-versions": "satisfied --skip-invalid --ignore \"prettier|angular|lit|sass|@storybook/web-components|@storybook/html|svelte|@testing-library|vue\"",
+    "check:installed-dependencies-versions": "satisfied --skip-invalid --ignore \"prettier|angular|lit|sass|@storybook/web-components|@storybook/html|svelte|@testing-library|vue|@cypress/react\"",
```

Comment:
`@cypress/react` has `next` as a peer dependency becuase it contains a plugin that should be used by nextjs. However we don't use it so the peer depedency in this case shouldn't be used